tests/test_modeling_common.py

```diff
@@ -542,6 +542,12 @@ def test_forward_signature(self):
                     else ["encoder_outputs"]
                 )
                 self.assertListEqual(arg_names[: len(expected_arg_names)], expected_arg_names)
+            elif model_class.__name__ in [*get_values(MODEL_FOR_BACKBONE_MAPPING_NAMES)] and self.has_attentions:
+                expected_arg_names = ["pixel_values", "output_hidden_states", "output_attentions", "return_dict"]
+                self.assertListEqual(arg_names, expected_arg_names)
+            elif model_class.__name__ in [*get_values(MODEL_FOR_BACKBONE_MAPPING_NAMES)] and not self.has_attentions:
+                expected_arg_names = ["pixel_values", "output_hidden_states", "return_dict"]
+                self.assertListEqual(arg_names, expected_arg_names)
```

Comment:
```suggestion
            elif model_class.__name__ in [*get_values(MODEL_FOR_BACKBONE_MAPPING_NAMES)]:
                expected_arg_names = ["pixel_values", "output_hidden_states", "return_dict"]
                if self.has_attentions:
                    expected_arg_names += ["output_attentions"]
                self.assertListEqual(arg_names, expected_arg_names)
```

Reply:
I tried to apply your suggestion, but then tests fail, because it adds `output_attentions` to the end of the expected args, whereas the last argument is `return_dict`.

Reply:
Ok !